packages/mui-styled-engine-sc/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/styled-engine-sc",
-  "version": "6.4.0",
+  "version": "6.4.2",
```

Comment:
Any idea why 6.4.1 was skipped?

Reply:
We probably just missed it in https://github.com/mui/material-ui/pull/45077.

The script sometimes skips packages. I'll take a look to check if there's something we can fix.

Reply:
We could use [`--force-publish`](https://github.com/lerna/lerna/tree/main/libs/commands/version#--force-publish) to trigger the update on unchanged packages as well, which is what we do to keep all packages in the same version regardless of changes. What do you think?